labrador/src/jl.rs

```diff
@@ -98,19 +107,27 @@ mod tests {
         let n = 10;
         let matrix = ProjectionMatrix::<4>::new(n);
 
-        assert_eq!(matrix.matrix.len(), 256, "Matrix should have 256 rows");
         assert_eq!(
-            matrix.matrix[0].len(),
+            matrix.matrix.len(),
+            PROJECTION_MATRIX_SIZE,
+            "Matrix should have 256 rows"
```

Comment:
Since you are using `PROJECTION_MATRIX_SIZE` instead of hardcoding the number can you also set dynamically the error message?
```suggestion
            "Matrix should have {} rows",
    PROJECTION_MATRIX_SIZE
```